internal/convenience/synchronizer_node/synchronizer_create.go

```diff
@@ -349,7 +63,7 @@ func (s SynchronizerCreateWorker) WatchNewInputs(stdCtx context.Context) error {
 					errCh <- ctx.Err()
 					return
 				default:
-					latestRawID, err = s.SyncInputCreation(ctx, latestRawID, page, abi)
+					err := s.SynchronizerCreateInput.SyncInputs(ctx)
```

Comment:
Naming things is hell: SynchronizerInputCreator
PS.: don’t worry, your name is fine.